spec/views/social_networking/profile_pages/show.html.erb_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+
+module SocialNetworking
+  RSpec.describe "social_networking/profile_pages/show.html.erb", type: :view do
+    fixtures :all
```

Comment:
Is there a reason you chose to mix doubles and real model objects? I'd suggest using only doubles here and getting rid of the fixtures because it will run faster, and the view only depends on readable attributes.